cmd/kops/create_cluster.go

```diff
@@ -103,6 +103,9 @@ type CreateClusterOptions struct {
 	// Egress configuration - FOR TESTING ONLY
 	Egress string
 
```

Comment:
You won't have any changes in this file at all, just remove them

Reply:
Yeah, it felt like featureflag + cli was overkill. I'll remove it